services/galley/src/Galley/API/Public.hs

```diff
@@ -458,6 +458,16 @@ sitemap = do
   mkFeatureGetAndPutRoute @'Public.TeamFeatureDigitalSignatures Teams.getDigitalSignaturesInternal Teams.setDigitalSignaturesInternal
   mkFeatureGetAndPutRoute @'Public.TeamFeatureAppLock Teams.getAppLockInternal Teams.setAppLockInternal
 
+  get "/teams/:tid/features/" (continue Teams.getAllFeaturesH) $
```

Comment:
```suggestion
  get "/teams/:tid/features" (continue Teams.getAllFeaturesH) $
```

(not sure it makes a difference, but this is how all other end-points in this module do it.)

Reply:
consistency is good

Reply:
doesn't look like this made it to develop?  @smatting and it looks like the trailing `/` is mandatory if you have it there.  m-|